File: C++/day1.cpp
#include <fstream>
#include <iostream>
#include <string>

using namespace std;

int main(){
    ifstream file("inputDay1.txt");

    int cont=1;
    string cadena,cadenaAux;
    getline(file,cadena,'\n');

    while(!file.eof()){
        getline(file,cadenaAux,'\n');
        if(cadena != "" && cadenaAux != ""){
            if(cadenaAux>cadena){
                cont++;
            }
        }
        cadena=cadenaAux;
    }

    cout << cont << endl;
    file.close();
}

File: Python/day1.py
f = open("inputDay1.txt", "r")

cont=1
linea = f.readline()

while linea != "":
    lineaAux = f.readline()

    if (lineaAux>linea):
        cont+=1

    linea=lineaAux

print(cont)
f.close()